### PR TITLE
feat: add filters to getAll()

### PR DESCRIPTION
**Pull request by TorstenDittmann**

### CODE REVIEW

File: tests/Runtimes/RuntimesTest.php [hunk 1]
```diff
@@ -194,7 +197,7 @@ public function testExecRuntimes()
             $value = \escapeshellarg((empty($value)) ? 'null' : $value);
             $value = "--env {$key}={$value}";
         });
-
+        Console::log('');
```
Comment (eldadfux) [inline tests/Runtimes/RuntimesTest.php:200]: For debugging?

Comment (TorstenDittmann) [reply tests/Runtimes/RuntimesTest.php:200]: Bildschirmfoto 2021-04-22 um 17 43 52
Actually to align the checkmarks 😅

**Review by eldadfux — APPROVED** [review]

**Comment by eldadfux** [comment]
Please re-trigger tests 🙏🏻

**Comment by eldadfux** [comment]
> Please re-trigger tests 🙏🏻

OK, fixed it :)

**Pull request merged**